t/commands/file/extract.t

```diff
@@ -0,0 +1,41 @@
+use strict;
+use warnings;
+
+use Cwd 'getcwd';
+my $cwd = getcwd;
+
+BEGIN {
+  use Test::More tests => 57;
+  use Data::Dumper;
+
+  use_ok 'Rex';
+  use_ok 'Rex::Commands::File';
+  use_ok 'Rex::Interface::Exec';
+  Rex::Commands::File->import;
+  Rex::Interface::Exec;->import;
+}
+
+my $tmp_dir = "/tmp";
+if ( $^O =~ m/^MSWin/ ) {
+  $tmp_dir = $ENV{TMP};
+}
+
+my $filename = "${tmp_dir}/test-$$.tgz";
```

Comment:
No problem, but Rex can be installed and run from Windows, just like almost every other CPAN distribution (or on Mac OS X, for that matter). And upon installing from CPAN it will run the tests and we'd like to avoid failing them. We can help modifying those parts and provide testing (and get lots of automated test outputs from CPAN testers.

The only important thing about this topic is that we can't assume that it will only run on Unix-like systems, so we try to use OS-agnostic, built-in file path manipulation tools like `File::Spec`.